CMakeLists.txt

```diff
@@ -10,6 +10,13 @@ set(PACKAGE "ccpp-framework")
 set(AUTHORS "Dom Heinzeller" "Grant Firl" "Mike Kavulich" "Steve Goldhaber")
 string(TIMESTAMP YEAR "%Y")
 
+#------------------------------------------------------------------------------
+# Set MPI flags for C/C++/Fortran with MPI F08 interface
+find_package(MPI REQUIRED C Fortran)
```

Comment:
We need to check if this works - in my experience, if you as for MPI REQUIRED C Fortran you also need to define LANGUAGES C Fortran in line 5. Better would be to remove the C dependency here, since there is no C code in ccpp-framework?